core/src/main/java/jenkins/formelementpath/FormElementPathPageDecorator.java

```diff
@@ -0,0 +1,20 @@
+package jenkins.formelementpath;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import hudson.Extension;
+import hudson.Main;
+import hudson.model.PageDecorator;
+import jenkins.util.SystemProperties;
+
+@Extension
+public class FormElementPathPageDecorator extends PageDecorator {
+
+    @SuppressFBWarnings("MS_SHOULD_BE_FINAL")
+    private static /*almost final */ boolean ENABLED = Main.isUnitTest ||
```

Comment:
I wonder if we should still be able to flip this switch via the installation of a plugin?
Currently we spin up some Jenkins' instance where we do not configure system properties (or doing so is hard) and the current flow is to install the `form-element-path` plugin without the use of the paths it provides. (e.g. auto provisioning in kubernetes)

@Vlatombe knows more about this area of the framework that we use internally.

Reply:
That's fine, we'll be able to provide the system property.

Reply:
Could also make it public as well

Reply:
as Vincent is ok with being able to provide it, we can keep it as it is for now (easier to open up, than restrict accesses later)

Reply:
I think we need to turn this off by default, even in tests, at least until it can be rewritten to not rely on DOM listeners: https://github.com/jenkinsci/bom/pull/746#issuecomment-985172193

Reply:
Ah that’s a shames :(

Reply:
⇒ #6004